src/Akka.Hosting/Logging/LoggingExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="LoggingExtensions.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using Akka.Event;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Akka.Hosting.Logging
+{
+    public static class LoggingExtensions
+    {
+        /// <summary>
+        /// Fluent interface to configure the Akka.NET logger system
+        /// </summary>
+        /// <param name="builder">The <see cref="AkkaConfigurationBuilder"/> being configured</param>
+        /// <param name="configurator">An action that can be used to modify the logging configuration</param>
+        /// <returns>The original <see cref="AkkaConfigurationBuilder"/> instance</returns>
+        public static AkkaConfigurationBuilder ConfigureLoggers(this AkkaConfigurationBuilder builder, Action<LoggerSetup> configurator)
+        {
+            var setup = new LoggerSetup(builder);
```

Comment:
Is there a way we can avoid taking a dependency on the `AkkaConfigurationBuilder` inside of here?

Reply:
I fixed this.